src/app/archive/archive.ml

```diff
@@ -10,12 +10,19 @@ let command_run =
      and log_level = Flag.Log.level
      and server_port = Flag.Port.Archive.server
      and postgres = Flag.Uri.Archive.postgres
+     and runtime_config_file =
+       flag "-config-file" (required string)
```

Comment:
Can this be optional? For example, it seems like we shouldn't need to pass the config file when restarting the archive process with an existing db.

Reply:
Good point, I'll make it optional